setup.py

```diff
@@ -8,4 +8,7 @@
       author='SLAC National Accelerator Laboratory',
       packages=find_packages(),
       description='IOC definitions for LCLS Beamline Devices',
-      )
+      entry_points={
+            'happi.containers': ['lcls = pcdsdevices.happi.containers'],
```

Comment:
What does the `lcls` mean here? 

I talk somewhat frequently about using entrypoints, but I've admittedly never used it myself - so excuse my ignorance.

Reply:
name... but seems useless for what we have... :( 
I think it would only matter in the case of a clash if someone else had a `happi.containers` entry with `lcls`. I tried if we could just use it without the `lcls = ` but no deal.

Reply:
Gotcha. How about making it the repo name by convention? `lcls` -> `pcdsdevices`

Reply:
Sounds good! Lemme fix that and rebase after the PR that you just merged.

Reply:
Fixed, rebased and force-pushed.